### DISCUSSION TITLE
Accept donations?

### DISCUSSION THREAD

**Discussion by solus-hq** [prose-im/prose-app-macos#138] [Q&A]
Hello!

It's a nicely designed and modern XMPP application which could be vital for our team since we're not willing to keep our chats in Telegram. 

I would like to donate to help this project grow. Hope you add some channel to accept cryptocurrency donations as well.

**Comment by valeriansaliou** [comment]
Hello! We could accept donations yes. Would a GH sponsor do? Thank you so much!

**Comment by solus-hq** [comment]
@valeriansaliou

I'm not sure if the GH program allows contributions in cryptocurrencies. You could simply post your BTC/LTC wallet in Readme.md =)

**Reply by solus-hq** [reply]
Just add wallet addresses since it's simple and convenient way to donate these days =)

**Comment by solus-hq** [comment]
Still willing to donate via BTC =)
